postcss.config.js

```diff
@@ -5,5 +5,12 @@ module.exports = ({ env }) => ({
     require('postcss-preset-env')({ stage: 1 }), // https://preset-env.cssdb.org/features#stage-1
     // Other plugins...
     env === 'production' ? require('cssnano')({ preset: 'default' }) : null, // How to use conditional plugins/options for prod
+    env === 'production'
```

Comment:
Thought: Does it make sense to perhaps enable purging in all envs? I.e. I'm thinking should the tooling unintentionally remove some CSS we'd not expected it to, we'd not notice during local development, and it'd ship to production where we might encounter a bug.

I've had a quick test of this locally, and it's very fast when it comes to rebuilding CSS even with the purge. What do you think?

Reply:
Yeah, makes sense. Better to spot any CSS issues locally while in dev rather than on the live production site.